services/workflows-service/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "workflow-service",
   "scripts": {
     "spellcheck": "cspell \"*\"",
-    "setup": "npm run docker:db && npm run db:reset && npm run db:migrate-dev && npm run seed",
+    "setup": "npm run docker:db:down && npm run docker:db && npm run db:reset && npm run seed",
```

Comment:
if there is no docker runnning, docker down will fail the script?

Reply:
will work as expected , it just do nothing